virttest/qemu_vm.py

```diff
@@ -1777,6 +1777,8 @@ def sort_key(dev):
                 serial_name = prefix + str(len(self.virtio_ports))\
                     if prefix else serial
                 serial_params['serial_name'] = serial_name
+            if serial_params['serial_type'] == 'pci-serial':
+                serial_params['serial_bus'] = self._get_pci_bus(serial_params, 'serial', False)
```

Comment:
This line is too long, otherwise LGTM.

Reply:
Thanks, updated, restrict to 80 character.